interchaintest/ibc_test.go

```diff
@@ -48,33 +51,36 @@ func TestRelayerInProcess(t *testing.T) {
 func TestRelayerDockerEventProcessor(t *testing.T) {
 	t.Parallel()
 
+	uuid := relayerinterchaintest.UniqueRelayerImageName()
+	relayerinterchaintest.BuildRelayerImage(t, uuid)
```

Comment:
`BuildRelayerImage` can return the `image` (or maybe `imageName`) to be passed to `CustomDockerImage`